SolovevaII/src/components/MessageField/MessageField.jsx

```diff
@@ -0,0 +1,29 @@
+import './MessageField.css';
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import  {Message, messageType} from "components/Message";
+
+export class MessageField extends Component{
+
+    static propTypes = {
+        list: PropTypes.arrayOf(
+            PropTypes.shape(messageType)
+        ),
+    };
+
+
+    render() {
+        const {list} = this.props;
+        console.log(list);
+
+        return (
+            <div className="message-field">
+                //Деструктуризация. Оператор «spread».
```

Comment:
Так нельзя писать комменты
```suggestion
                {/*Деструктуризация. Оператор «spread».*/}
```